chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -411,10 +411,16 @@ Zotero.Utilities.Internal = {
 		return s;
 	},
 	
-	isOnlyEmoji: function (str) {
-		// Remove emoji, Zero Width Joiner, and Variation Selector-16 and see if anything's left
-		const re = /\p{Extended_Pictographic}|\u200D|\uFE0F/gu;
-		return !str.replace(re, '');
+	containsEmoji: function (str) {
+		const re = /\p{Extended_Pictographic}/gu;
+		return !!str.match(re);
+	},
+
+	renderTagWithEmojis: function (str) {
```

Comment:
I agree - good point.

Reply:
Better idea! Something like `Zotero.Tags.extractEmojiForItemsList()`? No need to clutter Utilities.Internal.

Reply:
Makes sense! I think `containsEmoji` should stay with`extractEmojiForItemsList` because they are related, so in the last commit I suggest we move `containsEmoji` into `Zotero.Tags` too. 
Both of them are only used for tags, so it seems logical?

Reply:
No, I think `containsEmoji` should stay put. There's nothing tag-specific there, and it could easily be used in other contexts even if it's not now.

Reply:
Understood! I redid and force-pushed the last commit with only emoji extraction moved to `Zotero.Tags`